kustomize/commands/edit/add/configmap.go

```diff
@@ -36,7 +36,10 @@ func newCmdAddConfigMap(
 	kustomize edit add configmap my-configmap --from-env-file=env/path.env
 
 	# Adds a configmap from env-file with behavior merge
-	kustomize edit add configmap my-configmap --behavior=merge --from-env-file=env/path.env	
+	kustomize edit add configmap my-configmap --behavior=merge --from-env-file=env/path.env
+
+    # Adds a configmap to the kustomization file with a specific namespace
+    kustomize edit add configmap my-configmap --namespace test-ns --from-literal=my-key=my-value
```

Comment:
nit: 
![image](https://github.com/kubernetes-sigs/kustomize/assets/73322208/d3a269be-eece-4b86-8147-1f19aabf86d3)

Reply:
Tabs vs. spaces got me 😅
Thanks for spotting this 🙏 just fixed!